net/node/node.go

```diff
@@ -147,6 +146,7 @@ func NewNode() *node {
 	n := node{
 		state: INIT,
 	}
+	n.time = time.Now()
```

Comment:
Just a suggestion：
How about below？
```
	n := node{
 		state: INIT,
                time:  time.Now()，
 	}
```

Reply:
It didn't work, because time is a property of node.link but not node.

Reply:
My mistake. Sorry about that.